### PR TITLE
Add type declarations for material-ui-phone-number

### PR DESCRIPTION
**Pull request by nextglabs**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/material-ui-phone-number/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Type definitions for material-ui-phone-number 2.2
+// Project: https://github.com/alexplumb/material-ui-phone-number
+// Definitions by: Bassem (NextGLabs) <https://github.com/nextglabs>
+//                 Gelsoh <https://github.com/gelzoh>
+//                 Láďa Durchánek <https://github.com/durchanek>
+//                 Nifanic <https://github.com/nifanic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.1
```
Comment (peterblazejewicz) [inline types/material-ui-phone-number/index.d.ts:8]:
```suggestion
// Minimum TypeScript Version: 4.1
```
Why do you need this version? This limits possible interop.
The material-ui itself is at 3.8:
https://github.com/mui-org/material-ui/blob/master/package.json#L146

File: types/material-ui-phone-number/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+// Type definitions for material-ui-phone-number 2.2
+// Project: https://github.com/alexplumb/material-ui-phone-number
+// Definitions by: Bassem (NextGLabs) <https://github.com/nextglabs>
+//                 Gelsoh <https://github.com/gelzoh>
+//                 Láďa Durchánek <https://github.com/durchanek>
+//                 Nifanic <https://github.com/nifanic>
```
Comment (peterblazejewicz) [inline types/material-ui-phone-number/index.d.ts:6]: Why are you adding other handles? Those folks will be target of notifications, etc.
If included, you should probably explicitly mention them in PR description, so they could give ACK,
thx!

File: types/material-ui-phone-number/index.d.ts (hunk 1 above)
Comment (nextglabs) [reply types/material-ui-phone-number/index.d.ts]: Right, I changed it to 3.8 👍🏻

File: types/material-ui-phone-number/index.d.ts (hunk 2 above)
Comment (nextglabs) [reply types/material-ui-phone-number/index.d.ts]: I thought I should give these people some credits, but I removed them for now 👍🏻

**Review by peterblazejewicz — APPROVED** [review]
LGMT!
@nextglabs thx!

**Pull request merged**